data/anaconda.conf

```diff
@@ -76,6 +76,16 @@ enabled_repositories_from_treeinfo = addon optional variant
 # Enable installation from the closest mirror.
 enable_closest_mirror = True
 
+# Enable possibility to skip packages with conflicts and broken dependencies.
+#
+# This enables --ignorebroken parameter of the packages section and related DBus API.
+#
+# If this feature is disabled Anaconda won't start the installation when
+# --ignorebroken paramater is used. Instead print error message to user
+# when Anaconda is started and quit the installation process.
+# It will also block use of related DBus API.
```

Comment:
The first sentence it sufficient for this file. The remaining text is too detailed and can easily become obsolete. If you want to make sure that the feature for skipping broken packages is properly documented, I suggest to write a short paragraph in the Anaconda documentation. However, it is also documented in `pyanaconda.core.configuration.payload`, so it is not necessary.

Reply:
I agree with everything except the last sentence.

The point is that ``pyanaconda.core.configuration.payload`` is in Anaconda code so when someone wants to change just the configuration he/she don't know about this place.

So or so I will leave here only the first sentence, it seems enough to me.